.github/workflows/build.yml

```diff
@@ -44,5 +44,5 @@ jobs:
       - name: Upload EVE
         uses: actions/upload-artifact@v2
         with:
-          name: eve-${{ env.ARCH }}
-          path: eve.tar
+          name: lktcache
+          path: $HOME/.linuxkit/cache
```

Comment:
So what is the size of the cache expected @vmlemon ? We really need to make sure that the action remains fast and we don't abuse GH too much

Reply:
~/.linuxkit gets to 1.8GB, on my machine, at least, so I hope there's not a risk of blowing things up, on the GitHub side of the house.

Reply:
> ~/.linuxkit gets to 1.8GB, on my machine, at least

yeah but you have been building tons of images. In the case of what is in CI, it should be a small fraction.

Why not add a step to report on the size right before the upload?

Reply:
1.8GB is an absolute NO GO. In fact, I'm kind of getting frustrated that this new method wouldn't allow me to just upload what I need -- which is whatever layers belong to eve package. That sounds like a pretty oversight to me that needs to be fixed/worked around @deitch

Reply:
1.8GB is on your laptop. If you are running in a clean CI - which this is - you will get precisely the image parts, which should be the same as you would upload otherwise.

Reply:
The point is -- it is NOT just EVE -- and if your build rebuilds a lot of packages -- well, you know what that would be.

Reply:
Not sure if this is the best avenue to pursue, or if we'll eventually revert to the previous way of doing things, but I implemented the output, for the size of the cache, earlier on. The syntax of the config file doesn't seem massively intuitive, in terms of if you can add "run" steps, if there's "uses/with"-style stanzas, and have it actually function, though...

Reply:
> syntax of the config file

Do you mean the workflow yaml? No, they are not quite. :-)

> for the size of the cache

So what was it?

Reply:
> > syntax of the config file
> 
> Do you mean the workflow yaml? No, they are not quite. :-)
> 
> > for the size of the cache
> 
> So what was it?

Is there any way of actually testing the change, outside of the GitHub Actions environment? Short of nerfing my local LinuxKit cache, and repopulating it, I'm not sure how large a "good" one is supposed to be.